Tokamak Native/UIAppDelegate.swift

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019-2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Jed Fox on 07/01/2020.
+//
+
+import SwiftUI
+import UIKit
+
+// so we only need one Info.plist
+public class NSApplication: UIApplication {}
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+  let window = UIWindow()
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Tokamak Native/UIAppDelegate.swift;
  Line: 28;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 119 characters (`line_length`)